src/Compilers/CSharp/Portable/Symbols/Compilation_WellKnownMembers.cs

```diff
@@ -104,6 +104,11 @@ internal NamedTypeSymbol GetWellKnownType(WellKnownType type)
                 {
                     result = null;
                 }
+                else if (type == WellKnownType.System_Runtime_CompilerServices_NullableAttribute)
+                {
+                    // PROTOTYPE(NullableReferenceTypes): Handle consistently with other embedded attributes.
+                    result = ((SourceModuleSymbol)SourceModule).GetNullableAttribute();
```

Comment:
>GetNullableAttribute [](start = 64, length = 20)

What happens with use-site diagnostics if this is the first call for the GetNullableAttribute?

Reply:
Changed `SourceModuleSymbol` to store and return attribute and diagnostics.